dropshot/src/config.rs

```diff
@@ -43,12 +43,15 @@ pub type RawTlsConfig = rustls::ServerConfig;
 /// }
 /// ```
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
-#[serde(default)]
+#[serde(
+    from = "DeserializedConfigDropshot",
+    into = "DeserializedConfigDropshot"
```

Comment:
what's the point of the into?

Reply:
It is possible to derive `Serialize` on `ConfigDropshot` directly, and it would work today, but in case there's more divergence between the in-memory implementation and the serializable implementation over time, that may no longer hold true. I've seen this be a recurring source of bugs in other places, and making the types that directly interact with serde be the same for serialization and deserialization has always been helpful.